CODIGO/frmOpciones.frm

```diff
@@ -1701,8 +1676,8 @@ Private Sub scrVolume_Change()
     
     On Error GoTo scrVolume_Change_Err
     
-    Sound.VolumenActual = scrVolume.Value
-    VolFX = Sound.VolumenActual
+'    Sound.VolumenActual = scrVolume.Value
+'    VolFX = Sound.VolumenActual
```

Comment:
we need to be able to set fx volume, why remove VolFX = Sound.VolumenActual?
we should remove commented code

Reply:
resolved in next patch